R/fit.R

```diff
@@ -230,6 +230,7 @@ mmrm_control <- function(optimizer = stats::nlminb,
 #' @param optimizer (`string`)\cr optimizer to be used to generate the model.
 #' @param n_cores (`count`)\cr number of cores which could in principle be used for
 #'   parallel computations on Linux or Mac machines.
+#' @param ddfm (`string`)\cr denominator of degree of freedom method.
```

Comment:
```suggestion
#' @param method (`string`)\cr method used for adjusting degrees of freedom and covariance estimates.
```
or similar (easier name of argument, and it is not just for degrees of freedom)

Reply:
ok; I am just thinking we can make it consistent with SAS(sas use ddfm here)

Reply:
No need to be consistent with sas here